File: 0561-array-partition/0561-array-partition.java
class Solution {
    public int arrayPairSum(int[] nums) {
        // nums 배열 정렬
        Arrays.sort(nums);
        
        int sum = 0;
        // 앞에서부터 오름차순으로 반복
        for(int i = 0; i < nums.length; i++) {
            // 짝수번째일때 값의 합 계산
            if(i % 2 == 0) {
                sum += nums[i];
            }
        }
        
        return sum;
    }
}

File: 0561-array-partition/0561-array-partition.kt
class Solution {
    fun arrayPairSum(nums: IntArray): Int {
        var sum = 0
        nums.sort()
        
        // 앞에서부터 오름차순으로 인덱스 반복
        for((i, n) in nums.withIndex()) {
            // 짝수번째일때 합 계산
            if(i % 2 == 0) {
                sum += n
            }
        }
        
        return sum
    }
}